packages/desktop-client/src/components/budget/index.js

```diff
@@ -514,7 +517,13 @@ function BudgetWrapper(props) {
 
   let rolloverComponents = useMemo(
     () => ({
-      SummaryComponent: rollover.BudgetSummary,
+      SummaryComponent: props => (
```

Comment:
Please don’t pass a function expression like this as a component, since it will cause the component to be unmounted and remounted each time the parent component renders. (although the `useMemo` makes this less of a problem). Additionally, since the feature flags aren’t passed into the `useMemo` dependency list, so the component will not get the new value if the flags update while it is rendered.

Reply:
Good catch, fixed in 9c4b481.